vars/runStackPublishPipeline.groovy

```diff
@@ -21,20 +21,24 @@ def call() {
         }
 
         stages {
+            stage('Check Required Parameters') {
+                steps {
+                    sh "test -n '${params.version}' || ( echo 'The version is required. Please specify a version to release.' >&2 && exit 1)"
```

Comment:
Should we test for a leading v?

Reply:
I could go either way on that, but I ended up not doing it to have dumber test logic. I suppose I could always use a `test grep -q` though, which is fun

Reply:
Done. Test runs:

[Bad input](https://jenkinsci.upbound.io/job/upbound/job/suskin-test/job/master/15/console)
[Good input](https://jenkinsci.upbound.io/job/upbound/job/suskin-test/job/master/14/console)